src/pages/VeSDL/index.tsx

```diff
@@ -340,18 +336,22 @@ export default function VeSDL(): JSX.Element {
             <Typography>
               {t("totalVeSdlHolding")}: {formatUnits(veSdlTokenVal)}
             </Typography>
-            <Alert
-              severity="error"
-              icon={false}
-              sx={{
-                textAlign: "center",
-                display: !getPenalty() ? "none" : "block",
-              }}
-            >
-              {t("withdrawAlertMsg", { sdlValue: getPenalty() })}
-            </Alert>
+            {penaltyAmount && (
+              <Alert
+                severity="error"
+                icon={false}
+                sx={{
+                  textAlign: "center",
+                }}
+              >
+                {t("withdrawAlertMsg", {
+                  sdlValue: formatUnits(penaltyAmount),
+                })}
+              </Alert>
+            )}
             <Button
               variant="contained"
+              data-testid="unLockVeSdlBtn"
```

Comment:
small but unlock is one word

```suggestion
              data-testid="unlockVeSdlBtn"
```